scripts/azerite/README.md

```diff
@@ -0,0 +1,20 @@
+# Azerite Scaling Extractor
+
+This script pulls azerite scaling data from the simc repository (which must be
+cloned locally). It should *probably* be replaced by an implementation based on [this
+stuff](https://github.com/simulationcraft/simc/wiki/Using-CASC-Extract-and-DBC-Extract).
+
+Poke @emallson for questions/complaints.
+
+## Layout
+
+- `extract.js` -- Does the extraction from simc data.
+- `traits.json` -- A list of all relevant trait spell IDs. Note that *multiple
```

Comment:
Please mention that it's manually updated, and possibly explain how to add new ones.